concept/validation/schema-validation.feature

```diff
@@ -11,490 +11,734 @@ Feature: Schema validation
+    Then relation(rel1) get roles contain:
       | rel1:role1 |
-    Then relation(rel1) get related roles do not contain:
+    Then relation(rel1) get roles do not contain:
       | rel0:role0 |
     Then transaction commits
 
-    When session opens transaction of type: write
-    Then relation(rel1) set supertype: rel01; throws exception
+    When connection open schema transaction for database: typedb
+    Then relation(rel1) set supertype: rel01; fails
 
-    When session opens transaction of type: write
-    Then relation(rel00) unset related role: role00; throws exception
+    When transaction closes
+    When connection open schema transaction for database: typedb
+    Then relation(rel00) delete role: role00; fails
 
 
   Scenario: A type may only override an ownership it inherits
-    Given create attribute type: attr0, with value type: string
-    Given attribute(attr0) set abstract: true
-    Given create attribute type: attr1, with value type: string
+    Given create attribute type: attr0
+    Given attribute(attr0) set value type: string
+    Given attribute(attr0) set annotation: @abstract
+    Given create attribute type: attr1
     Given attribute(attr1) set supertype: attr0
     Given create entity type: ent00
-    Given entity(ent00) set abstract: true
-    Given entity(ent00) set owns attribute type: attr0
+    Given entity(ent00) set annotation: @abstract
+    Given entity(ent00) set owns: attr0
     Given create entity type: ent01
-    Given entity(ent01) set abstract: true
+    Given entity(ent01) set annotation: @abstract
     Given transaction commits
 
-    When session opens transaction of type: write
+    When connection open schema transaction for database: typedb
     When create entity type: ent1
-    Then entity(ent1) set owns attribute type: attr1 as attr0; throws exception
+    When entity(ent1) set owns: attr1
+    Then entity(ent1) get owns(attr1) set override: attr0; fails
 
-    When session opens transaction of type: write
+    When transaction closes
+    When connection open schema transaction for database: typedb
     When create entity type: ent1
     When entity(ent1) set supertype: ent00
-    When entity(ent1) set owns attribute type: attr1 as attr0
+    When entity(ent1) set owns: attr1
+    When entity(ent1) get owns(attr1) set override: attr0
     Then transaction commits
 
-    When session opens transaction of type: write
-    When entity(ent00) unset owns attribute type: attr0
-    Then transaction commits; throws exception
+    When connection open schema transaction for database: typedb
+    When entity(ent00) unset owns: attr0
+    Then transaction commits; fails
 
-    When session opens transaction of type: write
-    When entity(ent1) set supertype: ent01
-    Then transaction commits; throws exception
+    When connection open schema transaction for database: typedb
+    When entity(ent1) set supertype: ent01; fails
+
+
+  Scenario: A type may only override a plays it inherits
```

Comment:
I added this test following the idea of `owns` as it was missing. 
Should move to `plays.feature`
+ make a Scenario Outline with entity/relation